vyper.list.json

```diff
@@ -0,0 +1,193 @@
+{
+  "builds": [
+    {
+      "path": "https://github.com/vyperlang/vyper/releases/download/v0.3.6/vyper.0.3.6%2Bcommit.4a2124d0.linux",
+      "version": "vyper.0.3.6",
+      "prerelease": "",
```

Comment:
Nit: could we remove `prerelease` section for non-nightly binaries?

Reply:
Since there are no exposed nightly builds, I removed this field entirely